### ISSUE TITLE
Array.prototype.includes type definition issue

### ISSUE THREAD

**Issue by amerllica** [microsoft/TypeScript#48819]
# Bug Report

### 🔎 Search Terms

I saw a but in `Array.prototype.includes` type:

```
interface Array<T> {
    /**
     * Determines whether an array includes a certain element, returning true or false as appropriate.
     * @param searchElement The element to search for.
     * @param fromIndex The position in this array at which to begin searching for searchElement.
     */
    includes(searchElement: T, fromIndex?: number): boolean;
}
```
The above type definition comes from microsoft/TypeScript, and this is not a true generic type, you must not pass `T` to the `searchElement`, because the `searchElement` could be anything.

### 🕗 Version & Regression Information

version: 4.2.4

### 💻 Code

```typescript
type name = 'amer' | 'saeed' | 'ahmad';
const arr = ['amer', 'saeed'].includes(name); // here it returns this error, which is not correct
//// TS2345: Argument of type '"amer" | "saeed" | "ahmad"' is not assignable to parameter of type '"amer" | "saeed"'.   Type '"ahmad"' is not assignable to type '"amer" | "saeed"'.
```

### 🙁 Actual behavior

it returns a type error, told developer the type of argument in `includes` function must be the type of items in array.

### 🙂 Expected behavior

It should allow developer to insert anything in `includes` function.

**Comment by jcalz** [comment]
Duplicate #48247 and others.  

You forgot to fill out the “search terms” part of the template showing how you searched for existing issues before filing a new issue.

**Comment by amerllica** [comment]
I just closed this issue because of duplicated #48247
